src/include/migraphx/op/isnan.hpp

```diff
@@ -0,0 +1,45 @@
+#ifndef MIGRAPHX_GUARD_OPERATORS_ISNAN_HPP
+#define MIGRAPHX_GUARD_OPERATORS_ISNAN_HPP
+
+#include <migraphx/op/unary.hpp>
+#include <migraphx/config.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace op {
+
+struct isnan : unary<isnan>
+{
+    std::string point_function() const { return "isnan"; }
```

Comment:
This shouldn't be needed as `unary` will use operator name for the `point_function`.